packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,89 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
```

Comment:
Should this value we white or transparent? @neilpfeiffer

Reply:
My callouts had this listed as white. It's also listed as white [here](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.scss#L3) which I was told was the example I should be following.

Reply:
Sounds good. I wanted to double check because the button callouts also specified white, but it was intended to be transparent.

Reply:
#fff - white. match this:
https://github.com/cerner/terra-core/blob/8855edcf3197bbd029e9d4a61da21c3cf2059308/packages/terra-form-textarea/src/Textarea.scss#L3